measure_extinction/plotting/plot_ext.py

```diff
@@ -443,7 +447,10 @@ def plot_multi_extinction(
     plt.rc("axes.formatter", min_exponent=2)
 
     # create the plot
-    fig, ax = plt.subplots(figsize=(15, len(starpair_list) * 1.25))
+    ysize = 6
+    if spread:
+        ysize += len(starpair_list) * 1.25
+    fig, ax = plt.subplots(figsize=(8, ysize))
```

Comment:
Yeah.  It is clear that the plot size/ratio depends on the wavelength range being plotted.  When I tried to stack UV extinction curves, the figure was very oblong and not very useful.  I realized that this would muck up your work, but don't have any idea how to pick good defaults.  Maybe we should setup a set of data from different wavelengths for such testing?

Reply:
Thinking about it, I might actually already have a solution for this problem. In general, all the plotting settings will differ dependent on wavelength range, type of figure you want to make for your specific purpose, and even personal taste. I wanted to change settings (such as axis limits, tick labels, fontsize, etc.) for every plot in my paper, but it was too difficult to make every setting an optional keyword in the function. So, I just set the defaults in `plot_ext.py` to values that result in reasonably looking plots. And then, I added the line
`return fig, ax` at the end of every plot function (as you can see in the code), such that in my specific plotting scripts, I can further tweak the plots. I now found that I can also change the figure size after the plot was created, using:
`fig.set_figheight(20)` and `fig.set_figwidth(10)`.

So, long story short, we should just pick some default value for all settings and then tweak to our liking in our specific plotting scripts.

Reply:
Sounds like a plan.  Thanks for including the code on how to do this - will be very useful!